siip.tex

```diff
@@ -215,25 +224,26 @@
           \subsection{Kick-Off Workshop}
           A two-day kick-off workshop hosted at the University of Illinois at 
           Urbana-Champaign will allow the 
-          participants to sketch the initial framework of 
-          the collaboration and coordinate logistics. Through brainstorming and 
-          discussion activities, the workshop will deliver a number of 
-          products. These products will be published in an online repository 
-          and associated website, both hosted by GitHub. 
+          participants to brainstorm and discuss the scope of the project.
+          Among other potential products, the workshop will deliver
+          a sketch of the initial framework of the collaborative effort and
+          a logistical coordination of responsibilities. These products will
+          be published in an online GitHub repository 
+          and associated website.
 
           Invited participants in the first workshop will include UIUC PI Huff 
-          and UIUC Co-PI Davis, excternal collaborating professors Wilson, 
+          and UIUC Co-PI Davis, external collaborating professors Wilson, 
           Skutnik, Roberts, Scopatz, and Borrelli, and up to 2 UIUC professors 
-          who may be likely to extend this work in the future for their own 
-          courses.
+          who may be interested in adapting and extending this work for their own 
+          future courses.
 
-          As indicated in Table \ref{tab:kickoff}, products to be developed in the first workshop include learning 
-          objectives\cite{bloom_bloom_1984}, a concept 
-          map\cite{novak_concept_1990}, a list of fine-grained lesson topics, a 
-          directed acyclic graph describing dependencies among lessons. Additionally, concrete logistics 
+          As indicated in Table \ref{tab:kickoff}, products to be developed in
+          the first workshop include learning objectives\cite{bloom_bloom_1984}, a concept 
+          map\cite{novak_concept_1990}, an outline of fine-grained lesson modules, and a 
+          graph describing dependencies among lessons. Additionally, concrete logistics 
           for completion will be established, including individual curriculum 
-          development assignments, decisions to adopt specific raw formats 
-          for lesson content (markdown vs. \LaTeX, Jupyter vs MATLAB, etc.), 
+          development assignments; decisions to adopt specific raw formats 
```

Comment:
I don't think semicolons are intended to break sentences in this way. Each side should be a full independent clause.